src/ext/DifxApp/wixext/WixToolset.DifxApp.wixext.csproj

```diff
@@ -8,15 +8,13 @@
     <Description>WiX Toolset DIFxApp Extension</Description>
     <Title>WiX Toolset DIFxApp Extension</Title>
     <DebugType>embedded</DebugType>
+    <NuspecFile>$(MSBuildThisFileName).nuspec</NuspecFile>
   </PropertyGroup>
 
   <Import Project="..\..\WixExt.props" />
 
-  <ItemGroup>
-    <EmbeddedResource Include="$(OutputPath)..\difxapp.wixlib" />
-  </ItemGroup>
-
   <ItemGroup Condition=" '$(NCrunch)'=='' ">
-    <ProjectReference Include="..\wixlib\difxapp.wixproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" />
+    <ProjectReference Include="..\wixlib\difxapp.wixproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" Properties="Platform=x86" />
+    <ProjectReference Include="..\wixlib\difxapp.wixproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" Properties="Platform=x64" />
```

Comment:
Add the attribute `OutputItemType="Content"` to the `ProjectReference` and the `difxapp_$(Platform).wixlib` should be added.